src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -340,18 +342,65 @@ public Page Pop()
             return curTop;
         }
 
+        /// <summary>
+        /// Returns the index of the given page in Navigator.
```

Comment:
I think that GetPage does not return the index, but Page object for the given index - please check.

Reply:
Thank you for your comments.
I have fixed the summary.